.github/workflows/e2e-main.yml

```diff
@@ -104,6 +104,16 @@ jobs:
           PODMAN_DESKTOP_ARGS: ${{ github.workspace }}/podman-desktop
         run: yarn test:e2e
 
+      - name: Publish Test Report
+        uses: mikepenz/action-junit-report@v5
+        if: always() # always run even if the previous step fails
+        with:
+          fail_on_failure: true
```

Comment:
you are missing to set  `annotate_only: true` in both workflows. It will then not complain about insufficient permissions.

Reply:
I have just added it